packages/sdk-react/src/ui/FileViewer/FileViewer.tsx

```diff
@@ -26,90 +15,15 @@ export const SUPPORTED_MIME_TYPES = [
 ];
 
 export interface FileViewerProps {
-  /**
-   * Defines what PDF should be displayed.
-   * Its value can be a URL,
-   * a file (imported using import ... from ... or from file input form element),
-   * or an object with parameters
-   *  (
-   *   url - URL;
-   *   data - data, preferably Uint8Array;
-   *   range - PDFDataRangeTransport;
-   *   httpHeaders - custom request headers, e.g. for authorization
-   *   withCredentials - a boolean to indicate whether to include cookies in the request (defaults to false)
-   *  )
-   */
-  url?: string;
+  url: string;
   mimetype: string;
   name?: string;
-  rightIcon?: ReactNode;
-
-  /**
-   * What should be done when the PDF is not loaded and the user clicks the `Reload` button.
-   *
-   * Note: The button `Reload` will be displayed only if the `onErrorCallback` is provided
-   */
   onReloadCallback?: () => void;
 }
 
-export const FileViewer = (props: FileViewerProps) => {
-  const [reactPdfDynamic, setReactPdfDynamic] = useState<{
-    components: AsyncReturnType<typeof loadReactPDF> | null;
-    error: string | null;
-    loading: boolean;
-  }>({
-    error: null,
-    components: null,
-    loading: true,
-  });
-
-  const { i18n } = useLingui();
-
-  useEffect(() => {
-    /**
-     * We have to load react-pdf dynamically because it is not compatible with SSR.
-     */
-    loadReactPDF()
-      .then((components) => {
-        setReactPdfDynamic(() => ({
-          components,
-          error: null,
-          loading: false,
-        }));
-      })
-      .catch((error) => {
-        setReactPdfDynamic(() => {
-          const errorMessage = error.message;
-          return {
-            components: null,
-            error: errorMessage
-              ? t(i18n)`Failed to load PDF Viewer: ${errorMessage}`
-              : t(i18n)`Failed to load PDF Viewer`,
-            loading: false,
-          };
-        });
-      });
-  }, [i18n]);
-
-  if (reactPdfDynamic.loading) {
-    return <LoadingPage />;
-  }
-
-  if (reactPdfDynamic.components) {
-    const { Document, Page } = reactPdfDynamic.components;
-    return <FileViewerComponent {...props} Document={Document} Page={Page} />;
-  }
-
-  if (reactPdfDynamic.error) {
-    return (
-      <CenteredContentBox>
-        <Box color="danger">{reactPdfDynamic.error}</Box>
-      </CenteredContentBox>
-    );
-  }
-
-  return null;
-};
+export const FileViewer = (props: FileViewerProps) => (
+  <FileViewerComponent {...props} />
```

Comment:
please don't think I'm picking on trivia, but why do we need `<FileViewerComponent/>` then?